internal/jujuapi/modelmanager.go

```diff
@@ -124,7 +124,34 @@ func (r *controllerRoot) ListModelSummaries(ctx context.Context, _ jujuparams.Mo
 // ListModels returns the models that the authenticated user
 // has access to. The user parameter is ignored.
 func (r *controllerRoot) ListModels(ctx context.Context, _ jujuparams.Entity) (jujuparams.UserModelList, error) {
-	return r.allModels(ctx)
```

Comment:
now, we can either delete this `allModels` and `Model.ToJujuModel()` or you need to refactor the other consumer of this function.

Reply:
Follow up pr for all models, it's used in AllModels still, don't want to do all in this PR, was going to do it in the next.

Reply:
okok, perfect